docs/docusaurus/docs/cloud/connect/connect_snowflake.md

```diff
@@ -0,0 +1,214 @@
+
+<Tabs
+  groupId="connect-snowflake"
+  defaultValue='docker'
+  values={[
+  {label: 'Docker', value:'docker'},
+  {label: 'Kubernetes', value:'kubernetes'},
+  ]}>
+<TabItem value="docker">
+
+1. Download the GX Agent Docker container image from [Docker Hub](https://hub.docker.com/r/greatexpectations/agent).
+
+2. After configuring your cloud service to run Docker containers, run the following Docker command to start the GX Agent: 
+
+   ```bash title="Terminal input"
+   docker run -it \
+   -e GX_CLOUD_ACCESS_TOKEN= YOUR_ACCESS_TOKEN \ 
+   -e GX_CLOUD_ORGANIZATION_ID= YOUR_ORGANIZATION_ID \  
+   greatexpectations/agent:latest
+    ```
+    Replace `YOUR_ACCESS_TOKEN` and `YOUR_ORGANIZATION_ID` with the values you copied previously. To optionally store your Snowflake password as an environment variable, add `-e GX_CLOUD_SNOWFLAKE_PW= YOUR_SNOWFLAKE_PASSWORD \` to the Docker command.
+
+3. Run the following command to use the GX Agent image as the base image and optionally add custom commands:
+
+   ```bash title="Terminal input"
+   FROM greatexpectations/agent
+   RUN echo "custom_commands"
+   ```
+4. Optional. Run the following command to rebuild the Docker image:
+
+   ```bash title="Terminal input"
+   docker build -t myorg/agent
+   ```
+5. Optional. Run `docker ps` or open Docker Desktop to confirm the agent is running.
+
+</TabItem>
+<TabItem value="kubernetes">
+
+1. Install kubectl. See [Install Tools](https://kubernetes.io/docs/tasks/tools/).
+
+2. Run the following command to provide the access credentials to the Kubernetes container:
+    
+   ```sh
+   kubectl create secret generic gx-agent-secret \
+   --from-literal=GX_CLOUD_ORGANIZATION_ID=YOUR_ORGANIZATION_ID \
+   --from-literal=GX_CLOUD_ACCESS_TOKEN=YOUR_ACCESS_TOKEN \
+   ```
+    Replace `YOUR_ORGANIZATION_ID` and `YOUR_ACCESS_TOKEN` with the values you copied previously. To optionally include your Snowflake password as an access credential, add `--from-literal=GX_CLOUD_SNOWFLAKE_PW=YOUR_SNOWFLAKE_PASSWORD` to the command.
+
+
+3. Create and save a file named `deployment.yaml`, with the following configuration:
+
+   ```yaml
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+    name: gx-agent
+    labels:
+    app: gx-agent
+    spec:
+    replicas: 1
+    selector:
+    matchLabels:
+    app: gx-agent
+    template:
+    metadata:
+      labels:
+        app: gx-agent
+    spec:
+      containers:
+       name: gx-agent
+        image: greatexpectations/agent:latest
+        envFrom:
+        secretRef:
+         name: gx-agent-secret
+   ```
+4. Run the following command to use the `deployment.yaml`configuration file to deploy the GX Agent:
+
+   ```sh
+   kubectl apply -f deployment.yaml
+   ```
+5. Optional. Run the following command to confirm the agent is running:
+
+   ```sh
+   kubectl logs -l app=gx-agent
+   ```
+6. Run the following command to terminate running resources gracefully:
+
```

Comment:
If step 5 above is optional, then should step 6 be optional as well? I'll defer to you on this @kwcanuck

Reply:
@Shinnnyshinshin, that seems reasonable. I've added the edit.